From bf6db5b48ec25f83939f1fdebb59028bc3c40b00 Mon Sep 17 00:00:00 2001
From: "H. Peter Anvin (Intel)" <hpa@zytor.com>
Date: Wed, 6 Feb 2019 11:30:51 -0800
Subject: [PATCH] install: don't install obsolete file com32.ld

com32.ld has been obsolete for a long time, and has been removed now;
don't install it either.

Reported-by: Joakim Tjernlund <Joakim.Tjernlund@infinera.com>
Signed-off-by: H. Peter Anvin (Intel) <hpa@zytor.com>

Upstream-Status: Backport
Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 com32/lib/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/com32/lib/Makefile b/com32/lib/Makefile
index 74fff149..6a931492 100644
--- a/com32/lib/Makefile
+++ b/com32/lib/Makefile
@@ -113,7 +113,6 @@ spotless: clean
 
 install: all
 	mkdir -m 755 -p $(INSTALLROOT)$(COM32DIR)
-	install -m 644 $(SRC)/com32.ld $(INSTALLROOT)$(COM32DIR)
 	-rm -rf $(INSTALLROOT)$(COM32DIR)/include
 	cp -r $(SRC)/../include $(INSTALLROOT)$(COM32DIR)
 
-- 
2.17.1

